### DISCUSSION TITLE
DTU Configuration

### DISCUSSION THREAD

**Discussion by mirek-z** [tbnobody/OpenDTU#1193] [Q&A]
Hello
please help me, i'm trying to set up opendtu and i don't know what to enter in the field DTU Configuration: serial. is this supposed to be my dtu's serial number? because if so, there is not enough space and it does not accept letters :)

**Comment by tbnobody** [comment]
Yes it is the serial of the DTU. No you don't have to change anything. Just leave it as it is.

**Comment by stephan-cardot** [comment]
> Hello please help me, i'm trying to set up opendtu and i don't know what to enter in the field DTU Configuration: serial. is this supposed to be my dtu's serial number? because if so, there is not enough space and it does not accept letters :)

Hello, 
Did you find a solution to enter inverter serial number containing letters ?

**Comment by XatomicX01** [comment]
Hello,

same prolem.
My serial number contains letter : S/N : NNNNLNNNNLLN 
N for number and L for letter.
Is there a solution to add this inverter?
